### ISSUE TITLE
Can't auto-align Files app

### ISSUE THREAD

**Issue by decsolst** [vinceliuice/WhiteSur-gtk-theme#359]
I'm on Manjaro, kernel 5.10.42, and after installing this theme I cannot auto-align or "snap" the Files app, and as far as I know right now, _only_ the Files app. I've attached a screenshot of the expected behavior a window should have when dragged to one side of the screen, and the behavior the Files app exhibits. This is my first time using Github, so please feel free to let me know if I should include something more.
![fileswindow](https://user-images.githubusercontent.com/82539033/125264950-f9ab7780-e2c9-11eb-8085-b3e3769792c7.png)
![normalwindow](https://user-images.githubusercontent.com/82539033/125264957-fa440e00-e2c9-11eb-8d18-e30f316e2192.png)

**Comment by vinceliuice** [comment]
Snap to left or right is all ok

**Comment by rivanfebrian123** [comment]
@decsolst was right. Nautilus window snapping doesn't work on poor displays because it's too big

![Screenshot from 2021-07-12 17-27-44](https://user-images.githubusercontent.com/6300910/125272831-7d02b280-e336-11eb-860e-e202fdf69622.png)

**Comment by vinceliuice** [comment]
@rivanfebrian123 You are right, I testedon my laptop, all theme can not snap to left or right! (even adwaita) I don'tknow how to fix this, this seems not a theme issue

**Comment by rivanfebrian123** [comment]
The Adwaita theme actually works on my laptop. I think the problem is the headerbar (contents). Is it possible to make them smaller?

![Screenshot from 2021-07-12 17-49-15](https://user-images.githubusercontent.com/6300910/125279302-fc47b480-e33d-11eb-83d6-92f3502627a1.png)

WhiteSur, minimum width:

![Screenshot from 2021-07-12 18-23-13](https://user-images.githubusercontent.com/6300910/125279556-49c42180-e33e-11eb-9890-8fd9f6685885.png)

Adwaita, minimum width:

![Screenshot from 2021-07-12 18-22-54](https://user-images.githubusercontent.com/6300910/125279562-4b8de500-e33e-11eb-80fe-4c2b572226c8.png)

**Comment by decsolst** [comment]
Hm, I suspect this is related to the "nautilus sidebar minimum width" tweak @rivanfebrian123

**Comment by decsolst** [comment]
It seems that the minimum size the "-s" argument for install.sh will allow is 200. Anything less and it prompts me to change it to either the default or larger.

**Comment by vinceliuice** [comment]
Could you guys try the mojave style `./install.sh -N mojave`, I don't have a good computer to test, because all theme can not snap on my laptop (1920x1080 and 150% scale, but all theme works well on 125% scale and 100% scale)

**Comment by rivanfebrian123** [comment]
It works, Vince. So yeah, the problem is the headerbar

![Screenshot from 2021-07-12 20-58-42](https://user-images.githubusercontent.com/6300910/125300187-fd83dc00-e353-11eb-8bf7-e9d7f2b3b6e4.png)

![Screenshot from 2021-07-12 20-58-37](https://user-images.githubusercontent.com/6300910/125300196-feb50900-e353-11eb-933e-16a34b6c83cd.png)

**Comment by vinceliuice** [comment]
https://gitlab.gnome.org/GNOME/nautilus/-/issues/411
The nautius must can be resize the window to less than half of your resolution, then the snap can work well!

**Comment by vinceliuice** [comment]
I don't know if it works or not, I add a 180px version, could you guys try `./install.sh -s 180` for me?

**Comment by rivanfebrian123** [comment]
Here it is. I can't snap to the left

![Screenshot from 2021-07-12 22-57-38](https://user-images.githubusercontent.com/6300910/125319018-97ec1b80-e364-11eb-9c81-e39d95bbbdd1.png)

**Comment by vinceliuice** [comment]
You are right, the reason is the headerbar, 
![Screenshot from 2021-07-13 09-04-12](https://user-images.githubusercontent.com/7604295/125374361-6d787d80-e3b9-11eb-8d4b-cb7daded83fb.png)

**Comment by vinceliuice** [comment]
If I remove the headerbar left button's margin to the left, then it can snap!

**Comment by vinceliuice** [comment]
But if the margin removed, god! it'll looks very ugly!

**Comment by rivanfebrian123** [comment]
Yeah, that's ugly
